src/index.ts

```diff
@@ -25,7 +25,7 @@ const action = async (context: any): Promise<void> => {
 
     ({access_token} = JSON.parse(response.body));
   }
-  catch (err) {
+  catch {
     context.notify('There was a problem authenticating your account. Is your API info up to date?');
```

Comment:
Too bad the error is ignored here. The API response might have some more specific info.

@karaggeorge I think we need a way for plugins to log errors that are not shown to users, but that can help debugging.

Reply:
Working on that PR, coming in a few minutes

Reply:
In this case, what more can a user do than check their only inputs (API keys)? `got` automatically retries on failures, right? What could I offer the user from the error?

Reply:
There are many things that could go wrong other than the auth info being incorrect:

- The server could be down.
- The internet for the user could be down.
- The server could be overloaded.
- The API might have accidentally changed a parameter.
- So many more things.

---

> got automatically retries on failures, right? What could I offer the user from the error?

Not on authentication failure, but on other things, yes. But Got only retries 3 times. So if it doesn't work then, it would show the error.

Reply:
So you mean actually relay `errror.message` to the user in the notification?

Reply:
Yes, but, well, that's the problem. There's not enough space in the notification. So we need to find another way to show that information.